contracts/token/ERC777/README.adoc

```diff
@@ -18,3 +18,7 @@ Additionally there are interfaces used to develop contracts that react to token
 {{IERC777Sender}}
 
 {{IERC777Recipient}}
+
+== Extensions
+
+{{ERC777Mintable}}
```

Comment:
Can you add a note about Mintable in the opening paragraphs? Something like 

'Finally, the `ERC777Mintable` extension can be used to specify a set of addresses that are allowed to mint new tokens.'